src/models/user.rs

```diff
@@ -134,3 +135,16 @@ pub fn get_login_code(conn: &database::DbPool, email: &str) -> Result<i32, UserM
         _ => Err(UserModelError::MoreThanOneEmailError),
     }
 }
+
+pub fn get_id_by_email(conn: &database::DbPool, email: &str) -> Result<Uuid, UserModelError> {
```

Comment:
Same here about `Result<T>`. I will refrain from commenting again about this.

Reply:
Done.